apps/simple-trading-app/src/app/routes/router-config.tsx

```diff
@@ -0,0 +1,49 @@
+import { t } from '@vegaprotocol/react-helpers';
+import { DealTicketContainer } from '../components/deal-ticket';
+
+export const ROUTES = {
+  HOME: '/',
+  MARKETS: 'markets',
+  TRADING: 'trading',
+  LIQUIDITY: 'liquidity',
+  PORTFOLIO: 'portfolio',
+};
+
+export const routerConfig = [
+  {
+    path: ROUTES.HOME,
+    name: 'Home',
+    text: t('Home'),
+    element: <div>Home</div>,
+  },
+  {
+    path: ROUTES.MARKETS,
+    name: 'Markets',
+    text: t('Markets'),
+    element: <div>Markets</div>,
+  },
+  {
+    path: ROUTES.TRADING,
+    name: 'Trading',
+    text: t('Trading'),
+    element: (
+      <DealTicketContainer
+        marketId={
+          '41013c28d53a72225c07cf2660cdd415d9dd0e9317ec4574e77592332db35596'
```

Comment:
comment (non-blocking): as you probably know this will change next time testnet is restarted but I assume market select is coming in a new PR

Reply:
👍